src/dos/dos_keyboard_layout.cpp

```diff
@@ -649,37 +653,36 @@ Bit16u keyboard_layout::extract_codepage(const char* keyboard_file_name) {
 	auto tempfile = OpenDosboxFile(nbuf);
 	if (!tempfile) {
 		// try keyboard layout libraries next
-		if ((start_pos=read_kcl_file("keyboard.sys",keyboard_file_name,true))) {
-			tempfile = OpenDosboxFile("keyboard.sys");
-		} else if ((start_pos=read_kcl_file("keybrd2.sys",keyboard_file_name,true))) {
-			tempfile = OpenDosboxFile("keybrd2.sys");
-		} else if ((start_pos=read_kcl_file("keybrd3.sys",keyboard_file_name,true))) {
-			tempfile = OpenDosboxFile("keybrd3.sys");
-		} else if ((start_pos=read_kcl_file("keyboard.sys",keyboard_file_name,false))) {
-			tempfile = OpenDosboxFile("keyboard.sys");
-		} else if ((start_pos=read_kcl_file("keybrd2.sys",keyboard_file_name,false))) {
-			tempfile = OpenDosboxFile("keybrd2.sys");
-		} else if ((start_pos=read_kcl_file("keybrd3.sys",keyboard_file_name,false))) {
-			tempfile = OpenDosboxFile("keybrd3.sys");
-		} else if ((start_pos=read_kcl_data(layout_keyboardsys,33196,keyboard_file_name,true))) {
-			read_buf_size=0;
-			for (Bitu ct=start_pos+2; ct<33196; ct++) read_buf[read_buf_size++]=layout_keyboardsys[ct];
-		} else if ((start_pos=read_kcl_data(layout_keybrd2sys,25431,keyboard_file_name,true))) {
-			read_buf_size=0;
-			for (Bitu ct=start_pos+2; ct<25431; ct++) read_buf[read_buf_size++]=layout_keybrd2sys[ct];
-		} else if ((start_pos=read_kcl_data(layout_keybrd3sys,27122,keyboard_file_name,true))) {
-			read_buf_size=0;
-			for (Bitu ct=start_pos+2; ct<27122; ct++) read_buf[read_buf_size++]=layout_keybrd3sys[ct];
-		} else if ((start_pos=read_kcl_data(layout_keyboardsys,33196,keyboard_file_name,false))) {
-			read_buf_size=0;
-			for (Bitu ct=start_pos+2; ct<33196; ct++) read_buf[read_buf_size++]=layout_keyboardsys[ct];
-		} else if ((start_pos=read_kcl_data(layout_keybrd2sys,25431,keyboard_file_name,false))) {
-			read_buf_size=0;
-			for (Bitu ct=start_pos+2; ct<25431; ct++) read_buf[read_buf_size++]=layout_keybrd2sys[ct];
-		} else if ((start_pos=read_kcl_data(layout_keybrd3sys,27122,keyboard_file_name,false))) {
+		auto try_file = [&](const char* file, bool first_id_only) {
+			if (!(start_pos=read_kcl_file(file,keyboard_file_name,first_id_only)))
+				return false;
+			tempfile = OpenDosboxFile(file);
+			return true;
+		};
+		auto try_builtin = [&](uint8_t i, bool first_id_only) {
+			if (!(start_pos=read_kcl_data(BLOB_KEYBOARD_SYS[i],keyboard_file_name,first_id_only)))
+				return false;
```

Comment:
If you're on Linux / macOS / MSYS2 - the `./scripts/format-commit.sh` script will use clang-format to apply the project's formatting and will add spaces and help wrap some of these.

Just be sure to commit your changes first - then run the formatter (which formats the last commit), and you can pick and choose what you'd like to keep. 

@Wengier , for those using VisualStudio, do you have a way of applying the formatting?